alpa/__init__.py

```diff
@@ -3,14 +3,15 @@
 from alpa.device_mesh import (DeviceCluster, PhysicalDeviceMesh,
                               LocalPhysicalDeviceMesh,
                               DistributedPhysicalDeviceMesh, DistributedArray,
-                              fetch)
+                              ReplicatedDistributedArray, fetch)
```

Comment:
do you use `ReplicatedDistributedArray` somewhere?

Reply:
I use it in the `serialization.py`.